### ISSUE TITLE
boundary_mask feature

### ISSUE THREAD

**Issue by snownus** [kyamagu/paperdoll#1]
Hi,

It seems that the boundary mask features are the same for all the images after the image normalization. Then, what about the main function of this boundary mask feature ?  Could you help address the issue?

**Comment by kyamagu** [comment]
@snownus Yes, the boundary mask is a bias for indicating how close to the border of the image. Thus, the same for all the images after normalization.
